webapp/i18n/ja.json

```diff
@@ -2012,6 +2054,7 @@
   "user.settings.security.ldap": "AD/LDAP",
   "user.settings.security.loginGitlab": "GitLabでログインしました",
   "user.settings.security.loginLdap": "AD/LDAPでログインする",
+  "user.settings.security.loginSaml": "GitLabでログインしました",
```

Comment:
Thanks @enahum! I see it.
I don't think this PR should be rejected. Please merge this.

 I wanna suggest that translation PR don't include FUZZY translation when making translation PR in the future, because some incorrect translation from system user will be merged to master. 
I think that untranslated messages is better than incorrect translations.

But if it's time-consuming work, traslation PR should be done as has been the way until now.

Thank you.